tests/tests.js

```diff
@@ -59,6 +59,21 @@ describe("htmlbars-inline-precompile", function() {
     }, /placeholders inside a tagged template string are not supported/);
   });
 
+  it("moduleName is supplies to the precompiler", function() {
+    var fullpath = path.join(__dirname, 'fixtures', 'hello-world.js');
+
+    var precompile = function(template, options) {
+      assert.equal(options.moduleName, fullpath);
```

Comment:
Did some digging, it looks like `sourceRoot` only knocks off the path when dealing with module transformations.

Would you be for adding an optional second arg that has a precompilerOptions method who's return value is used to pass on to the precompiler?

```js
HTMLBarsInlinePrecompilePlugin(precompile, {
  precompileOptions(opts) {
    // example of how someone might use it
    var sourceRootRegEx = new RegExp("^" + opts.sourceRoot + "/?");

    return {
      moduleName: opts.filenameRelative.replace(sourceRootRegEx, "")
    };
  }
});
```


Not implementing the method will have the same behavior as today.